docs/README.md

```diff
@@ -85,6 +85,16 @@ We have so far discussed a few different strategies to solve the rendering probl
 
 and it's not just contentState but contentState + some config
 
+## Support
+
+### Python support
+Python versions as defined in `setup.py` classifiers.
```

Comment:
Ah! I didn't know there were Markdown linters around. I'll update it then, I'm all for anything which prevent diffs getting to big/unreadable just because style differences.

That being said, if the linter configuration doesn't live in the repo like `.editorconfig` or `setup.cfg` do, it might be hard to enforce properly.